src/index.ts

```diff
@@ -241,7 +247,7 @@ const transform = async ({
                   expression.callee?.type === "MemberExpression" &&
                   expression.callee.object?.type === "AssignmentExpression" &&
                   expression.callee.object.left?.object?.name === "self" &&
-                  expression.callee.object.left.property?.value ===
+                  expression.callee.object.left.property?.name ===
```

Comment:
(mostly note to self 😅)

I've checked this and I believe it's correct, the expression is:
`(self.webpackChunk_N_E = self.webpackChunk_N_E || []).push([[...], {...`

so `webpackChunk_N_E`'s indeed a prop name there

<details>

![Screenshot 2023-03-10 at 13 59 35](https://user-images.githubusercontent.com/61631103/224336077-9be570eb-191a-4b98-a99e-fb8573698e3e.png)


</details>